junit-jupiter-api/src/main/java/org/junit/jupiter/api/condition/DisabledOnOs.java

```diff
@@ -24,10 +24,11 @@
 /**
  * {@code @DisabledOnOs} is used to signal that the annotated test class or
  * test method is <em>disabled</em> on one or more specified
- * {@linkplain #value operating systems}.
+ * {@linkplain #value operating systems} or on one or more specified
+ * {@linkplain #architectures architectures}
  *
  * <p>When applied at the class level, all test methods within that class
- * will be disabled on the same specified operating systems.
+ * will be disabled on the same specified operating systems or architectures.
```

Comment:
We need to clearly document AND vs. OR semantics, potentially in a dedicated paragraph.

Reply:
how about "will be enabled/disabled on the same specified combination of operating systems and architectures."?

Reply:
> We need to clearly document AND vs. OR semantics, potentially in a dedicated paragraph.

I might even be worth a clarification in the user guide?